tryffle/uploaded_pdf/views.py

```diff
@@ -4,19 +4,29 @@
 from django.conf import settings
 from uploaded_pdf.models import DocumentPdf, Page
 from inertia import render as inertia_render
+from .serializers import DocumentPdfSerializer, PageSerializer
+
 # Create your views here.
 def documents(request):   
     document = DocumentPdf.objects.all()
```

Comment:
Si cette vue fonctionne, la vue avec le moteur de recherche ressemblera à celle là, sauf qu'il y aura des filtres à passer (_path/to/search/api/?search=truc_) et il faudra aller fouiller dans request pour retrouver le _truc_ que tu veux.

Après, il faudra travailler la queryset DocumentPdf.objects.filter(...).all()